mozilla/voice-web
# Add Open Voice Challenge option to the info page

This commit updates privacy and mailing list options on the
/profile/info page.

<img width="456" alt="Screen Shot 2019-10-15 at 5 13 26 PM" src="https://user-images.githubusercontent.com/1840854/66878900-e0e52500-ef70-11e9-9d73-5d6bb1240050.png">

NOTE: As communicated in the TODOs, this commit is not ready for
production.

Test plan:

- Remove the `false && ...` condition from /web/src/components/pages/profile/info/info.tsx
- Visit localhost:9000/en/profile/info

The new option should appear, with new styles.
